GPUDay.Theme/package.json

```diff
@@ -46,5 +46,20 @@
       "target": "wwwroot/css"
     }
   },
-  "packageManager": "pnpm@10.8.0+sha512.0e82714d1b5b43c74610193cb20734897c1d00de89d0e18420aebc5977fa13d780a9cb05734624e81ebd81cc876cd464794850641c48b9544326b5622ca29971"
```

Comment:
This is causing a mismatch. We should keep it on the default. Then, it can be decided under https://github.com/Lombiq/NodeJs-Extensions/issues/116 whether we want to do this.